requirements.txt

```diff
@@ -1,13 +1,11 @@
-certifi==2018.4.16
-chardet==3.0.4
-Django==1.11
-future==0.16.0
-idna==2.6
-mock==2.0.0
-pbr==4.0.2
-pycryptodomex==3.6.1
-pyjwkest==1.4.0
-pytz==2018.4
-requests==2.18.4
-six==1.11.0
-urllib3==1.22
+certifi>=2022.12.7
+chardet>=5.1.0
+Django==3.2.18
+idna>=3
+mock>=5
+pbr>=5
+pycryptodome>=3.17
+pyjwkest>=1.4.0
+pytz>=2023.3
+requests>=2.29.0
+urllib3>=1.26.15
```

Comment:
tried to match what chassis will use so that were working with a common `base`.
drop python 2 support.